test/cql-pytest/util.py

```diff
@@ -14,6 +14,10 @@
 import collections
 from contextlib import contextmanager
 
+from cassandra.auth import PlainTextAuthProvider
+from cassandra.cluster import Cluster, ConsistencyLevel, ExecutionProfile, EXEC_PROFILE_DEFAULT
+from cassandra.policies import RoundRobinPolicy
```

Comment:
Most of these imports were prevbiously in conftest.py, so you can remove them from contest.py in this patch.